guides/source/developers/taxation/overview.html.md

```diff
@@ -49,7 +49,7 @@ In Solidus, a tax rate consists of at least four values:
 
 - The descriptive name for the tax rate. For example, "Minnesota Sales Tax" for
   a Minnesota state tax rate.
-- The zone that the tax rate should apply to. <!-- TODO: Link to zones article.-->
+- The [zone](../locations/zones.html) that the tax rate should apply to.
```

Comment:
Can we use the link format at the end of the paragraph as the rest of guides pages (I think). Should be something like:

```md
...
The [zone][zone] that the tax rate should apply to.
...

[zone]: ../locations/zones.html
```

Reply:
Sure. 👍